colorsheet/build.gradle

```diff
@@ -72,15 +72,15 @@ android {
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
 
-    implementation 'androidx.appcompat:appcompat:1.1.0-alpha04'
-    implementation 'com.google.android.material:material:1.1.0-alpha05'
-    implementation "androidx.core:core-ktx:1.1.0-alpha05"
-    implementation "androidx.constraintlayout:constraintlayout:2.0.0-alpha3"
-
-    testImplementation 'junit:junit:4.12'
-    androidTestImplementation 'androidx.test:runner:1.1.1'
-    androidTestImplementation 'androidx.test.espresso:espresso-core:3.1.1'
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation 'androidx.appcompat:appcompat:1.2.0-alpha02'
```

Comment:
We can use the latest stable versions instead of alpha/beta

Reply:
Noted, will make the change.